src/ProjectTemplates/Microsoft.Extensions.AI.Templates/Microsoft.Extensions.AI.Templates.csproj

```diff
@@ -46,7 +46,7 @@
   <ItemGroup>
     <!-- Keep the exclude patterns below in sync with those in AIChatWebSnapshotTests.cs -->
     <Content
-      Include="src\ChatWithCustomData\**\*"
+      Include="src\*\**\*"
```

Comment:
I think this will bundle into the one package.  You may want diff icons for this template to use the MCP logo instead of the chat logo.